code/modules/jobs/job_types/lawyer.dm

```diff
@@ -29,6 +29,9 @@
 
 	job_tone = "objection"
 
+// Because we love our firebird-correct lawyer references, every other
+// lawyer will be dressed in purple, rather than blue.
+GLOBAL_VAR_INIT(lawyers_equipped, 0)
```

Comment:
Should be a static var instead, no need to pollute GLOB even more

Reply:
I was under the (maybe mistaken) impression that static vars were a bit too magical, and/or just a global that was hidden from people. Using a global var rather than static felt more "honest" if that makes sense.

Happy to change it back though. :bow:

Reply:
Static vars are fine and generally encouraged over GLOB pollution